inference-engine/tests/functional/shared_test_classes/src/single_layer/region_yolo.cpp

```diff
@@ -42,8 +45,11 @@ void RegionYoloLayerTest::SetUp() {
     int end_axis;
     InferenceEngine::Precision netPrecision;
     std::tie(inputShape, classes, coords, num_regions, do_softmax, mask, start_axis, end_axis, netPrecision, targetDevice) = this->GetParam();
+
+    init_input_shapes({ inputShape });
+
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
-    auto param = std::make_shared<ngraph::op::Parameter>(ngPrc, inputShape);
+    auto param = std::make_shared<ngraph::op::Parameter>(ngPrc, inputShape.first);
```

Comment:
You can use builder for parameter with dynamic shape. [Example](https://github.com/openvinotoolkit/openvino/blob/021477055387dadcd001021138b116b7880c3469/inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp#L77)

Reply:
updated, thanks. Some notes: 1) original source code used is this approach 2) to test support engineers: bad practice to have not really neccessary (`inputDynamicShapes`) base type members